doc/extensions/acrbuildext/azext_acrbuildext/build.py

```diff
@@ -221,6 +218,11 @@ def acr_queue(cmd,
     if source_location is None:
         source_location = "."
 
+    tar_file_path = os.path.join(tempfile.gettempdir(),
+                                 "source_archive_{}.tar.gz".format(hash(os.times())))
+
+    isLocalFile = True
```

Comment:
`is_local_file`, you might be able to move this assignment into the following `if` block.